src/posix/platform/openthread-system.h

```diff
@@ -35,6 +35,8 @@
 #ifndef OPENTHREAD_SYSTEM_H_
 #define OPENTHREAD_SYSTEM_H_
 
+#include "platform-config.h"
```

Comment:
Is this necessary? This file is exported as a public header file.

Reply:
Good point. Did not notcie that we expose this as public header for the posix-app plat lib. We currently don't need/use any config option in this file. I think for future we can follow the same model as OT public headers (i.e. do not use any `#if SOME_CONFIG` checks in public headers).
I will change the commit such that "platform-config.h" is only included form private source files under posix-app platform.